certified-connectors/Delinea Secret Server/apiProperties.json

```diff
@@ -0,0 +1,57 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "baseurl": {
+        "type": "string",
+        "format": "hostname",
```

Comment:
@tylerezimmerman one of our checks is failing because of the `baseurl` object. I think the `format` property causes the problem. Can you please try removing this property to see if the swagger check passes for the PR? I think it doesn't match the schema used for swagger check.